NagyAdrian/teme/3.3-obiecte/script.js

```diff
@@ -0,0 +1,59 @@
+// Scrieți o funcție care creează un obiect pentru a păstra informații despre rețeta ta preferată
+
+// Input: titlu = “Pizza”, portii = 3, ingrediente = [“sunca”, “cas”, “ketchup”]
+
+// Output: { titlu: “Pizza”, portii: 3, ingrediente: [”sunca”, “cas”, “ketchup”] }
+
+let reteta = { titlu: "", portii: "", ingredinte: [] }
+function Scriere(o) {
+  o.titlu = "pizza"
+  o.portii = "3"
+  o.ingrediente = ["sunca", "cas", "ketcup"]
+}
+
+Scriere(reteta)
+
+console.log(reteta)
+
+
+
+//Scrieți o funcție care returnează un obiect nou fără proprietățile furnizate
+
+//Input: obiect = { a: 1, b: 2 }, proprietate = “b”
+
+//Output: { a: 1 }
+
+const obiect = { a: 1, b: 2 }
+function StergerePropritetate(o) {
+  delete obiect.b
+}
+StergerePropritetate(obiect)
```

Comment:
![image](https://user-images.githubusercontent.com/45266485/201029489-72ebcc5a-104a-4c60-b289-7b89455ed38b.png)
- Trebuie refacut complet. 
- Denumirea functiei e o imbunatatire fata de prima instanta. Dar folosim camelCase pentru denumire, si din nou, as insista sa fie in engleza.